conda_forge_tick/auto_tick.py

```diff
@@ -752,7 +753,7 @@ def migrator_status(
         ]
         if pr_json and "PR" in pr_json:
             # I needed to fake some PRs they don't have html_urls though
-            node_metadata["pr_url"] = pr_json["PR"].get("html_url", "")
+            node_metadata["pr_url"] = pr_json["PR"].get(feedstock_url( fctx=feedstock_ctx, protocol="https"), "") 
```

Comment:
Please remove the space after the paren (we use black for code formatting but I don't think we've ran it recently so I don't want you to use it and blow up your PR) also I think you will need to trim the .git at the end of the resulting string

Reply:
ok and I can delete that comment too?

Reply:
> Please remove the space after the paren (we use black for code formatting but I don't think we've ran it recently so I don't want you to use it and blow up your PR) also I think you will need to trim the .git at the end of the resulting string

@CJ-Wright do you mind if I set a pre-commit hooks and some GH actions here to check for black? That way we'll be better equipped to review GSoC PRs.

Reply:
Please feel free, I think @mariusvniekerk has put some stuff up in prep for that